tests/integration/defs/disaggregated/test_disaggregated.py

```
@@ -2856,8 +2860,22 @@ async def _send_mixed_request(session,
-    except Exception:
-        pass  # connection abort on cancel is expected
+            elif not should_cancel and not completed:
+                # Server closed the stream without a valid completion
+                # (e.g. kv_transfer_timeout cancellation, error response).
+                result["server_rejected"] = True
+                result[
+                    "rejection_detail"] = f"http_{resp.status}_incomplete_stream"
+    except asyncio.TimeoutError:
+        # 120s aiohttp client timeout: server never responded in time.
+        result["timed_out"] = True
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)